src/routes/Chat/component.js

```diff
@@ -197,24 +191,6 @@ export default class Chat extends Component {
 				</Screen.Content>
 				{ !livechat_kill_switch ? (
 					<Screen.Footer
-						options={options ? (
-							<FooterOptions>
-								<Menu.Group>
-									{onPrintTranscript && !store.state.hidePrint && (
-										<Menu.Item onClick={onPrintTranscript} icon={PrintIcon}>{I18n.t('Print Chat')}</Menu.Item>
-									)}
-									{onChangeDepartment && (
-										<Menu.Item onClick={onChangeDepartment} icon={ChangeIcon}>{I18n.t('Change department')}</Menu.Item>
-									)}
-									{onRemoveUserData && (
-										<Menu.Item onClick={onRemoveUserData} icon={RemoveIcon}>{I18n.t('Forget/Remove my data')}</Menu.Item>
-									)}
```

Comment:
@AlexanderKanakis @ear-dev FYI This removes the `Forget/Remove my data` feature. Which we are not using at the moment. Just keep a note for the future.